packages/workflow/src/NodeHelpers.ts

```diff
@@ -1255,14 +1265,13 @@ export const tryToParseArray = (value: unknown): unknown[] => {
 };
 
 export const tryToParseObject = (value: unknown): object => {
+	if (value && typeof value === 'object') {
```

Comment:
Nit: `typeof value === 'object'` should be enough since `typeof undefined === 'undefined'`

Reply:
Yeah, but typescript is complaining without this check.